examples/vision/md/randaugment.md

```diff
@@ -0,0 +1,421 @@
+# RandAugment for Image Classification for Improved Robustness
+
+**Author:** [Sayak Paul](https://twitter.com/RisingSayak)<br>
+**Date created:** 2021/03/13<br>
+**Last modified:** 2021/03/13<br>
+**Description:** RandAugment for training an image classification model with improved robustness.
```

Comment:
Suggestion: "robustness" can imply many things IMHO, let's be specific and use "translational invariance of convolutional neural networks" (conv nets suffer from this problem as we've seen works, such as Capsule Nets that are trying to address this).

```suggestion
**Description:** Implementing RandAugment to improve translational invariance during image classification.
```

Reply:
Actually, RandAugment's efficacy is beyond just translational invariance. It also helps to increase robustness to common corruptions and perturbation. Hence the general usage of the word.